src/components/Form/FormUtils.js

```diff
@@ -90,25 +90,19 @@ export const dateParser = (date) => {
 };
 
 export const dayOfWeek = {
-                        0: "sunday",
-                        1: "monday",
-                        2: "tuesday",
-                        3: "wednesday",
-                        4: "thursday",
-                        5: "friday",
-                        6: "saturday",
-                    };
-
-export const weeklySessionsParser = (startDate, endDate) =>{
-    let start = new Date(startDate);
-    let end = new Date(endDate);
-    return Math.floor((end.getTime() - start.getTime()) / (1000 * 3600 * 24 * 7)) + 1;
+    "0": "sunday",
+    "1": "monday",
+    "2": "tuesday",
+    "3": "wednesday",
+    "4": "thursday",
+    "5": "friday",
+    "6": "saturday",
 };
 
 export const weeklySessionsParser = (startDate, endDate) => {
     const start = new Date(startDate);
     const end = new Date(endDate);
-    return Math.floor((end.getTime() - start.getTime()) / (1000 * 3600 * 24 * 7)) + 1;
+    return Math.floor((end.getTime() - start.getTime()) / (1000 * 3600 * 24 * 7));
```

Comment:
maybe? not too sure tbh. i was getting an extra session because of the +1, so i just subtracted 1... thing with this one is that the time zone shouldn't matter since we're looking for the duration/weekly sessions